modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/CalciteQueryProcessorTest.java

```diff
@@ -1216,6 +1216,35 @@ public void testInfixTypeCast() {
         assertThat(qCur.fieldsMeta().get(3).fieldTypeName(), equalTo(String.class.getName()));
     }
 
+    @Test
+    public void quantifiedCompTest() throws InterruptedException {
+        populateTables();
+
+        assertQuery(client, "select salary from account where salary > SOME (10, 11) ORDER BY salary")
+            .returns(11d)
+            .returns(12d)
+            .returns(13d)
+            .returns(13d)
+            .returns(13d)
+            .check();
+
+        assertQuery(client, "select salary from account where salary < SOME (12, 12) ORDER BY salary")
+            .returns(10d)
+            .returns(11d)
+            .check();
+
+        assertQuery(client, "select salary from account where salary < ANY (11, 12) ORDER BY salary")
+            .returns(10d)
+            .returns(11d)
+            .check();
+
+        assertQuery(client, "select salary from account where salary > ANY (12, 13) ORDER BY salary")
```

Comment:
`ANY` AFAIK is a synonym for SOME, but what about `ALL` operator? Looks like it also supported now, but there are no tests for `ALL`